system/lib/emmalloc.c

```diff
@@ -56,9 +58,6 @@
 // Behavior of right shifting a signed integer is compiler implementation defined.
 static_assert((((int32_t)0x80000000U) >> 31) == -1, "This malloc implementation requires that right-shifting a signed integer produces a sign-extending (arithmetic) shift!");
```

Comment:
does static_assert work in C? (looks like down below there is `_Static_assert`, maybe they should agree?

Reply:
It compiles.. so it must work I guess.

Reply:
https://en.cppreference.com/w/c/language/_Static_assert:

"This keyword is also available as convenience macro static_assert, available in the header <assert.h>"

Reply:
I made them agree on `static_assert`